dev-docs/bidder-adapter-1.md

```diff
@@ -358,6 +357,39 @@ function createBid(status, reqBid, response) {
 
 {% endhighlight %}
 
+#### If you would like to participate in auctions for outstream video ads
+
+Prebid.js passes a signal on video impressions that indicates whether the video context is instream or outstream.  You can start ingesting this signal if you wish.
+
+As described in [Show Outstream Video Ads]({{site.baseurl}}/dev-docs/show-outstream-video-ads.html), video ad units have a publisher-defined video context, which can be either `"instream"` or `"outstream"`:
+
+```javascript
+...
+mediaTypes: {
+    video: {
+        context: "outstream"
+    },
+},
+...
+```
+
+For an ad unit to play outstream ads, a "renderer" is required.  A renderer is a client-side media player (a JS file) that can play the VAST creative from the bid response.
```

Comment:
The underlying concept of a "renderer" is more generic.

In general, a renderer is the client-side code (usually a combination of JavaScript, HTML and CSS) responsible for displaying a creative on a page. Fundamentally, a renderer for outstream ads must provide a player environment capable of playing a video creative (most commonly, a VAST XML document)